website/theme.config.tsx

```diff
@@ -53,3 +53,33 @@ export default defineConfig({
   },
   titleSuffix: ` – ${SITE_NAME}`,
 })
+
+function YogaLogo({ className }: { className?: string }) {
```

Comment:
Just use Yoga logo from alpha versions of components

Reply:
Since https://github.com/the-guild-org/the-guild-components/pull/510 contains an update for the Codegen logo too, we'd have to update all tools websites to use the alpha release to keep the logos consistent.

Let's first agree about the change, keeping the logo here allows for faster testing of potential variations too.